modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -62,3 +62,22 @@ The index partitions residing on the recovered node would be reused during a rec
 3. Perform the software update or hardware maintenance operations.
 4. Perform recover rebalance operation.
 5. Cluster is back to normal/pre-failover safe state.
+
+=== Why rebalance button remains active even after a successful rebalance
+
+Even after a succesful Search rebalance operation, if the rebalance button remains `active` then it hints that the Search cluster is not having enough nodes
+to support the number of replicas configured for the Search index. The Search service considers the cluster is still imbalanced as it isn't having enough
+hardware resources to support the set configuration of replica count for the index partitions.
+
+Though validation checks ensures that there are enough search nodes to support the configured number of `replica` partitions, the number of search nodes
+could get compromised during the lifespan of a cluster due to various unforeseen events like hardware/network failures. And the `active` rebalance button issue surface if the subsequent recovery or node addition rebalance operations for Search service isn't bringing in enough nodes to support the configured replica count
+for the partitions.
+
+The user could resolve this by either,
+
+- adding enough Search nodes to the cluster to support the configured `replica` count.
+- editing the index definition(s) to adjust the replica count plausible in the cluster.
+
+Note - The maximum number of replica count feasible in a cluster is one less than the total number of Search nodes in the cluster.
```

Comment:
The preview of the note on line 81 is not rendered properly like the first note that's there in the same file.